api/proto/metrics/metrics.proto

```diff
@@ -432,6 +432,8 @@ message newlogMetric {
   logfileMetrics appMetrics = 18;
   // top 10 device log (not app) source in total bytes in percentage
   map <string, uint32> top10_input_sources = 19;
+  // gzip files removed due to exceeding user defined quota
```

Comment:
Does this count the files which fall off the end of the circular buffer? If so the count will always increase which wouldn't be indicate of any loss of log information.

Reply:
the files removed from 'keepSentQueue' is not counted in this, only the unsent is counted

Reply:
Makes sense updating the comment in that case.